src/test/java/lotto/model/LottoResultCalculatorTest.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoResultCalculatorTest {
+    @Test
+    @DisplayName("2등 당첨을 포함하지 않는 로또 당첨 결과를 계산한다.")
+    void getLottosResult() {
+        Lottos lottos = new Lottos(
+                Arrays.asList(
```

Comment:
```suggestion
                List.of(
```
위와 같이 구현도 가능함
Arrays.asList()와 List.of() 중 어느 api를 사용하는 것이 더 좋을까?
ai를 통해 확인해 본다.

Reply:
앗.. 이 부분 기본적인 건데 놓쳤네요 😂